src/Artsy/Router/Components/StorybookRelay.tsx

```diff
@@ -0,0 +1,37 @@
+import React from "react"
+import { QueryRenderer } from "react-relay"
+import { Environment, RecordSource, Store } from "relay-runtime"
+import { createMockNetworkLayer } from "../../Relay/createMockNetworkLayer"
+
+export const StorybookRelay = ({
+  Component,
+  query,
+  mockResolvers,
+}: {
+  Component: React.ComponentClass | React.SFC
+  query: any
+  mockResolvers: { [typeName: string]: () => any }
+}) => {
+  const network = createMockNetworkLayer({
+    Query: () => ({}),
+    ...mockResolvers,
+  })
+  const source = new RecordSource()
+  const store = new Store(source)
+  const environment = new Environment({
+    network,
+    store,
+  })
+
+  return (
+    <QueryRenderer
+      // tslint:disable-next-line
```

Comment:
What is this for? Can it be reduced to disabling a single named rule (and maybe just for 1 line)?

Reply:
Ah I meant to ask you about this! It's a lint rule you wrote:

https://github.com/artsy/reaction/blob/6b33396f3aaa2a11423455df2a294704c1019e4d/tslint/relayOperationGenericsRule.js#L64

Does it make sense to ignore it in this context as a one-off, or is the rule not doing what's intended?

Reply:
Ah I see. Yeah ignoring it here makes sense, but narrow it down to just this rule and only for this line.